config/sets/sylius/price-history/up-sylius-1.13-with-price-history-plugin-to-sylius-1.13.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\Namespace_\RenameNamespaceRector;
+use Rector\Removing\Rector\Class_\RemoveTraitUseRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(RenameNamespaceRector::class, [
```

Comment:
Shouldn't it be:
```suggestion
    $rectorConfig->ruleWithConfiguration(RenameClassRector::class, [
```
?

Reply:
If I see correctly, we need RenameNamespace in this case if I understand correctly 🤔 
https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#renamenamespacerector
And you can see the test for it [here](https://github.com/Sylius/SyliusRector/pull/23/files#diff-3b5a256c70e42fe32772e7c5aec55f08ef2fe49a4692a9943ba747b56f4533f2)